examples/external-api-request/external_api_request.py

```diff
@@ -23,16 +23,15 @@
 class ApiCallService(Service):
     @staticmethod
     async def get_payload():
-        manifest = open("manifest.json", "rb").read()
-        manifest = base64.b64encode(manifest).decode("utf-8")
+        manifest = open("manifest.json.base64", "rb").read().decode("utf-8")
```

Comment:
Should we add checks if this file (and maybe the files below) exists and is in the correct format (base64)?